demos-europe/demosplan-core
# fix: update dplan to EDT 0.14.0 and fix event problems

Some changes were needed to use EDT 0.14.0 in dplan. But while testing it in EWM I've also found some addon-refactoring-related bugs, which I've fixed here.

### How to review/test


What you could do:

You could check the EDT changelog and verify that these changes are the only ones needed.

I've tested the segment list in EWM, you could do additional testing in EWM or other projects, especially in places that use the generic API.

You could also compare check if the code in its current state makes sense.

### Linked PRs (optional)


### Tasks (optional)


### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
